crate/rsl-gui/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "rsl-gui"
+version = "6.0.13"
+authors = [
+    "Matthew Kirby <xoparr@gmail.com>",
+    "Fenhl <fenhl@fenhl.net>"
+]
+edition = "2018"
+license = "MIT"
+readme = "../../README.md"
+
+[dependencies]
+futures = "0.3"
+itertools = "0.10"
+rfd = "0.1"
+semver = "0.11"
+serde_json = "1"
+smart-default = "0.6"
+tokio-stream = "0.1"
+
+[dependencies.derive_more]
+version = "0.99"
+default-features = false
+features = ["from"]
+
+[dependencies.enum-iterator]
+git = "https://github.com/dasgefolge/enum-iterator"
+branch = "fields" #TODO finish this PR
+
+[dependencies.iced]
+git = "https://github.com/hecrj/iced" #TODO go back to the version on crates.io once it's compatible with tokio 1
```

Comment:
Two TODO's here. Are these still relevant?

Reply:
The first one is a reference to <https://github.com/stephaneyfx/enum-iterator/pull/7> which @leiftw and I still need to finish. The second is currently waiting on a new [iced](https://docs.rs/iced) release — that crate has historically been slow to push changes from GitHub to crates.io, but the maintainer [plans to release more frequently](https://github.com/hecrj/iced/issues/706#issuecomment-763718669).